submissions/fabunmiTolulope/javascript/conditionals.js

```diff
@@ -0,0 +1,25 @@
+// The positive difference between 7 and 123 is 116 so this program will print numbers between 1 and 116
+let number1 = 7;
+let number2 = 123;
+positiveDiff = number2 - number1;
+// prog is the programmed variable
+
+for (var prog = 1; prog <= positiveDiff; prog++) 
+{
+	if (prog% 3 === 0 && prog% 5 === 0) 
```

Comment:
the `prog` and `%` are joined together. kindly separate it

Reply:
@adaugochi I have seperated them.